packages/util/src/sha1.ts

```diff
@@ -103,19 +103,19 @@ export class Sha1 {
   /**
    * Internal compress helper function.
    * @param {!number[]|!Uint8Array|string} buf Block to compress.
-   * @param {number=} opt_offset Offset of the block in the buffer.
+   * @param {number=} optOffset Offset of the block in the buffer.
    * @private
    */
-  compress_(buf, opt_offset?) {
-    if (!opt_offset) {
-      opt_offset = 0;
+  compress_(buf, optOffset?): void {
```

Comment:
Should just be `offset` if we are following Google style guide (see previous comment)